pinot-hadoop/src/main/java/com/linkedin/pinot/hadoop/job/JobConfigConstants.java

```diff
@@ -21,4 +21,11 @@
   public static final String PATH_TO_OUTPUT = "path.to.output";
 
   public static final String TARGZ = ".tar.gz";
+
```

Comment:
Can you clean up the constants from all hadoop jobs (`SegmentCreation, SegmentTarPush, SegmentUriPush`)? We use some of constants from this class and we use hard coded values for some other constants (e.g. `push.to.host`, `push.to.port`)